yw_website/yw_website/apps/website/templates/includeable/my_workflow_list.html

```diff
@@ -0,0 +1,77 @@
+{% load static %}
+
+{% comment %}
+ <!-- Content to be paginated must be assigned to the workflow_list keyword argument when including
+        
+    Example:
+    {% include 'includeable/paginator.html' with workflow_list=<pageable_object> %}
+
+
+    The 'workflow_list' object must be a list like object with each 'workflow' element 
+    having the following attributes.
+
+    workflow:
+        title       {string}
+        description {string}
+        graph       {string}    /* graphviz string representation */
+        version_id  {int}
+ -->
+{% endcomment %}
+
+
+{% if workflow_list %}
+
+    <ol id="elementStyle">
+        <script type="text/javascript" src={% static "scripts/home_render_options.js" %}></script>
+
+        {% for workflow in workflow_list %}
+            <li>
+                <div classs = "tags">
+                    Tags: {{ workflow.tags | join:", " }}
+                </div>
+                <div class = "title">
+                    {{ workflow.title|linebreaks }}
+                    <form method="POST" action=" {% url 'edit_page' workflow.id workflow.version_id %}" style="align-self: right; margin-right: 2%; border-radius: 10px;">
+                        {% csrf_token %}
+                        <button type="submit"> Edit</button>
+                    </form>
+                </div>
+                <div class = "last_modified"> 
+                    {{ workflow.version_modified }}
+                </div>
+
+                <div id="graph{{ forloop.counter }}" class = workflow-image></div>
+
+                <div class = "workflow-details" id="description">
+                    <b>Description: </b>{{ workflow.description }}
+                </div>
+
+                <div style = "text-align:center">
+                    <a class = "more-details-button" href="{% url 'detailed_workflow' workflow.id workflow.version_id %}"> View Details</a>
+                </div>      
+                <!-- <button class="details-button" onClick = "description_view()">View Description</button> -->
+            </li>
+            <script>
+                function description_view(){
+                    var description = document.getElementById("description");
+                    if (description.style.display === "none") {
+                        description.style.display = "block";
+                    }
+                    else {
+                        description.style.display = "none";
+                    }
+                }
+            </script>
+            <script type="text/javascript">
+                graph = `{{ workflow.graph|safe }}`
+            </script>
+            <script type="text/javascript" src={% static "scripts/graphviz_render.js" %}></script>
+        {% endfor %}
+
+    </ol>
+
+{% else %}
+
```

Comment:
I think a block should be added to this includable that can be overrided in the my_workflow_list template so we don't have an extra file to maintain, if we make changes to the list views

Reply:
I 100% agree.